### ISSUE TITLE
Is all identifiers relevant to show

### ISSUE THREAD

**Issue by MortenHofft** [gbif/hp-grscicoll#158]
https://grscicoll.hp.gbif-staging.org/institution/0e919a55-08d3-4bd2-aec1-200858fafd92

Some institution and collection pages have tons of identifiers. Is it relevant to show all of them or can some be removed? If so should it be done in data or as a filter in the UI

**Comment by MortenHofft** [comment]
For example is it really relevant to show past `GRSCICOLL URI` values?

**Comment by ManonGros** [comment]
Yes I agree and I have been wanting to log an issue on handling identifiers (give context for them, arranging the display) since I got feedback last December! (late I know)
Thanks for opening the issue.

**Comment by ManonGros** [comment]
I propose to display only the primary identifier (https://github.com/gbif/registry/issues/625) or if none is selected, to show the entry URL as preferred identifier.

I am thinking that it could be something like "Cite the institution/collection: " + the preferred identifier (or URL if none is selected)
And a "other identifiers" and then either a collapsable list of the identifiers or a link to the registry with all the identifiers.

What do you think @MortenHofft ?

**Comment by MortenHofft** [comment]
That sounds reasonable. Other places we propose an acutla citation string and not just an identifier. But I would think it was easier to use just the identifier (I imagine you publish in another language or there is restrictions on citation formats).

But perhaps we should also add the GBIF key as an identifier. Based solely on https://github.com/gbif/portal-feedback/issues/5873#issuecomment-2870487259
